### ISSUE TITLE
Whether can log record time

### ISSUE THREAD

**Issue by sohaha** [kzhovn/statusbar-pomo-obsidian#35]
By the way, can terminate the operation of the time can also be logged

**Comment by kzhovn** [comment]
I'm not sure what you're asking. Can you rephrase this?

**Comment by sohaha** [comment]
support for recording start time

**Comment by sohaha** [comment]
Support for recording start time

**Comment by rob-hamill** [comment]
@kzhovn Maybe I missed it, but how do I log the duration of the session? (Or alternatively the start and end date of the session?). The doc includes below statement but don't think that answers this question.

 By default, the log message is "🍅 dddd, MMMM DD YYYY, h:mm A" (e.g. "🍅 Friday, July 16 2021, 6:18 PM"), but you can specify your own message using [moment display format](https://momentjs.com/docs/#/displaying/format/).
